File: algo/Python/BubbleSort.py
class BubbleSort:
    def iterative(arr):
        for i in range(len(arr)):
            for j in range(len(arr)):
                if arr[i] < arr[j]:
                    arr[i], arr[j] = arr[j],arr[i]
        return arr

arr = [53,21,52,14,63,31,52]
print(BubbleSort.iterative(arr))

File: algo/Java/Sorting/BubbleSort.java
package algo.Java.Sorting;

public class BubbleSort{
    public int[] iterative(int [] arr){
        
        for(int i=0;i<arr.length - 1;i++){
            for(int j=0;j<arr.length;j++){
                int temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;

            }
        }

        return arr;
    }

    // public int[] recursive(int [] arr){
        
    //     int arr;
    // }
}